src/legacy/core_plugins/kibana/public/discover/index.html

```diff
@@ -122,6 +90,15 @@ <h1 id="kui_local_breadcrumb" class="kuiLocalBreadcrumb" ng-if="opts.savedSearch
               </span>
             </button>
 
+            <div class="dscResultCount">
+              <span data-test-subj="discoverQueryHits" class="kuiLocalBreadcrumb__emphasis">{{(hits || 0) | number:0}}</span>
+              <span
+                i18n-id="kbn.discover.hitsPluralTitle"
+                i18n-default-message="{hits, plural, one {hit} other {hits}}"
+                i18n-values="{ hits }"
+              ></span>
+            </div>
```

Comment:
Can you add the refresh/reset button to this container and center the contents like so:

<img width="620" alt="Screen Shot 2019-07-19 at 15 13 19 PM" src="https://user-images.githubusercontent.com/549577/61559535-e72e3880-aa37-11e9-8203-2855ccb1d066.png">

Reply:
The original button was relevant only to saved searches.
I made the button show only for saved searches in the location you suggested.

Did you mean to also move the main reset button (the blue one in the search bar)? If so, I think it should be a part of a wider redesign :)